.github/actions/linux-build/action.yml

```diff
@@ -0,0 +1,73 @@
+name: "Linux Build"
+description: "Build the C++ library on Linux."
+inputs:
+  config: 
+    description: "Configuration to build (Release or Debug, see CMAKE_BUILD_TYPE)"
+    required: true
+    default: "Release"
+  compiler:
+    description: "Compiler to use (clang or gcc)."
+    required: true
+    default: "gcc"
+  version:
+    description: "Version of the compiler to use (min or latest)."
+    required: true
+    default: "latest"
+  coverage:
+    description: "Turn coverage on (ON or OFF)"
+    required: false
+    default: "OFF"
+  sanitize-ub:
+    description: "Turn on UB sanitzer (ON or OFF)"
+    required: false
+    default: "OFF"
+  sanitize-addr:
+    description: "Turn on address sanitzer (ON or OFF)"
+    required: false
+    default: "OFF"
+runs:
+  using: "composite"
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get -qq update
+        sudo apt-get -qq -y install libhdf5-dev lcov
+    - name: Build
+      shell: bash
+      run: |
+        cd cpp
+        if [[ "${{ inputs.compiler }}" == "gcc" ]]; then
+          if [[ "${{ inputs.version }}" == "min" ]]; then
+            export CC=/usr/bin/gcc-7
```

Comment:
which version of 7 is this then? At least we should mention it in a comment to say like "gcc-7.x tested"

Reply:
It's 7.5, that's what's preinstalled on the github runners. I think it's not useful in a comment, but it should be in the build instruction docs.